test/functional/example_test.py

```diff
@@ -34,7 +35,7 @@
 # message is received from the node-under-test. Subclass NodeConnCB and
 # override the on_*() methods if you need custom behaviour.
 class BaseNode(NodeConnCB):
-    def __init__(self):
+    def __init__(self, dstaddr, dstport, net="regtest", services=NODE_NETWORK, send_version=True):
```

Comment:
In commit "[tests] Make NodeConnCB a subclass of NodeConn"

Can get rid of all these default parameters repeated so many times. See main comment.